src/scripts/main.js

```diff
@@ -1,3 +1,43 @@
 'use strict';
 
-// write your code here
+const rows = document.querySelectorAll('tr');
+
+const headerCells = document.querySelectorAll('thead th');
+const secondHeaderCopy = headerCells[1].textContent;
+
+const newHeaderCell = document.createElement('th');
+
+newHeaderCell.textContent = secondHeaderCopy;
+
+const lastHeaderCell = headerCells[headerCells.length - 1];
+
+lastHeaderCell.insertAdjacentElement('beforebegin', newHeaderCell);
+
+rows.forEach((row, index) => {
+  if (index === 0 || index === rows.length - 1) {
+    return;
+  }
+
+  const secondRow = row.querySelectorAll('td')[1];
```

Comment:
Ensure that there are at least two cells in each row before attempting to access `row.querySelectorAll('td')[1]`. This will prevent potential errors if the table structure is not as expected.